proto_parser/src/proto_parser.cc

```diff
@@ -148,8 +148,90 @@ std::unique_ptr<ProtoValue> ProtoParser::CreatePrimitive(
     int repeated_field_index, 
     const google::protobuf::FieldDescriptor* field_descriptor, 
     int indent_count) {
-  // TODO(alexanderlin): Implement.
-  return nullptr;
+  const google::protobuf::Reflection* reflection = message.GetReflection();
+  // String used for GetRepeatedStringReference.
+  std::string str;
```

Comment:
Move string str to where it is using. you can do it by

case XX: {
  string str;
  ...

}